src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -766,6 +766,10 @@ def load_arguments(self, _):
         c.argument('connection_name', options_list=['--name', '-n'], help='Name of the peering connection.', id_part='child_name_2')
         c.argument('peer_circuit', help='Name or ID of the peer ExpressRoute circuit.', validator=validate_er_peer_circuit)
 
+    with self.argument_context('network express-route peering connection list') as c:
+        c.argument('circuit_name', id_part=None)
```

Comment:
Why is 'id_part=None' needed?

Reply:
Because we are not suppose to add `--ids` argument for `list` and `create` commands.